_test_common.sh

```diff
@@ -69,7 +69,7 @@ function assert_equals {
 	fi
 }
 
-function main {
+function display_test_location {
 	_TEST_RESULT="true"
```

Comment:
@davisaints, no need to create this new function since we're just defining a new variable. Please do:
```
function main {
    LIFERAY_RELEASE_TEST_MODE="true"
    _TEST_RESULT="true"
   ...
}
```